### PR TITLE
CMake updates and fixes

### PR DESCRIPTION
**Pull request by nmoinvaz**

### CODE REVIEW

**Comment by sergio-nsk** [comment]
Is not it considered as distribution of 3rd party executable binaries?

**Comment by sergio-nsk** [comment]
https://partner.steamgames.com/documentation/sdk_access_agreement

> 1.1 License Grant. Valve hereby grants Licensee a nonexclusive, royalty-free, terminable, worldwide, nontransferable license to:
> 
> <...>
> 
> (b) reproduce and distribute the part of the SDK provided inside the folder named `redistributable_bin` (the "SDK Redistributables") along with the Licensee Software in object code form.

It seems ok.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The steam interop example shows how to get information from the Steamworks API.
 
 ### Requirements
 
-* CMake 2.8
+* CMake 3.0
```
Comment (sergio-nsk) [inline README.md:16]: I suppose `bin - Steam shared libraries` above should be `bin - Steamworks  shared libraries` for consistency with `steamworks - Steamworks headers`.

**Review by sergio-nsk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nmoinvaz) [reply README.md:16]: Fixed.

**Review by sergio-nsk — APPROVED** [review]

**Pull request merged**